### PR TITLE
ET-889 - formatting

### PR DESCRIPTION
**Pull request by cliffordp**
[ET-889] ... but not really... no artifact needed

[ET-889]: https://moderntribe.atlassian.net/browse/ET-889

### CODE REVIEW

File: src/Tribe/Editor/Template/Overwrite.php [hunk 1]
```diff
@@ -191,34 +194,40 @@ public function has_tickets_support( $post_id ) {
...
-		if ( ! ( $this->ticket_type instanceof Tribe__Tickets__Tickets ) ) {
+		if ( ! $this->ticket_type instanceof Tribe__Tickets__Tickets ) {
 			return;
 		}
 
 		$ticket_form_hook = $this->ticket_type->get_ticket_form_hook();
 		if ( ! empty( $ticket_form_hook ) ) {
-			remove_action( $ticket_form_hook, array(
+			remove_action(
+				$ticket_form_hook, [
```
Comment (tr1b0t) [inline src/Tribe/Editor/Template/Overwrite.php:221]: :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: d3f7d5c0b309dab2a89228ba7917b583</sub>

File: src/Tribe/Editor/Template/Overwrite.php [hunk 2]
```diff
@@ -191,34 +194,40 @@ public function has_tickets_support( $post_id ) {
...
 				$this->ticket_type,
 				'maybe_add_front_end_tickets_form',
-			), 5 );
+			], 5 );
```
Comment (tr1b0t) [inline src/Tribe/Editor/Template/Overwrite.php:224]: * :no_entry_sign: Line indented incorrectly; expected at least 4 tabs, found 3
* :no_entry_sign: Array closer not aligned correctly; expected 16 space(s) but found 12
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 256d0b5f5a1b6dbe5623403285c613e5</sub>

File: src/Tribe/Editor/Template/Overwrite.php [hunk 3]
```diff
@@ -191,34 +194,40 @@ public function has_tickets_support( $post_id ) {
...
 		}
 
-		remove_filter( 'the_content', array(
+		remove_filter(
+			'the_content', [
```
Comment (tr1b0t) [inline src/Tribe/Editor/Template/Overwrite.php:228]: :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 7564c22918f5401e4ff28c2f181fba93</sub>

File: src/Tribe/Editor/Template/Overwrite.php [hunk 4]
```diff
@@ -191,34 +194,40 @@ public function has_tickets_support( $post_id ) {
...
 			$this->ticket_type,
 			'front_end_tickets_form_in_content',
-		), 11 );
+		], 11 );
```
Comment (tr1b0t) [inline src/Tribe/Editor/Template/Overwrite.php:231]: * :no_entry_sign: Line indented incorrectly; expected at least 3 tabs, found 2
* :no_entry_sign: Array closer not aligned correctly; expected 12 space(s) but found 8
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 981b3fe65eb29bc749752c759c5fdedd</sub>

File: src/Tribe/Editor/Compatibility/Tickets.php [hunk 5]
```diff
@@ -45,20 +48,30 @@ public function include_frontend_form( $content = '' ) {
 		}
 
 		// We don't care about anything other than event for now
-		if ( class_exists( 'Tribe__Events__Main' ) && Tribe__Events__Main::POSTTYPE !== $post->post_type ) {
+		if (
+			class_exists( 'Tribe__Events__Main' )
+			&& Tribe__Events__Main::POSTTYPE !== $post->post_type
```
Comment (sc0ttkclark) [inline src/Tribe/Editor/Compatibility/Tickets.php:53]:
```suggestion
			&& defined( 'Tribe__Events__Main::POSTTYPE' )
			&& Tribe__Events__Main::POSTTYPE !== $post->post_type
```

**Comment by sc0ttkclark** [comment]
Thx for keeping formatting in it's own PR

**Pull request merged**